src/java.base/share/classes/java/util/jar/JarInputStream.java

```diff
@@ -167,10 +167,21 @@ public JarEntry getNextJarEntry() throws IOException {
     }
 
     /**
-     * Reads from the current JAR file entry into an array of bytes.
-     * If {@code len} is not zero, the method
-     * blocks until some input is available; otherwise, no
-     * bytes are read and {@code 0} is returned.
+     * Reads from the current ZIP entry into an array of bytes, returning the number of
```

Comment:
Please change "Reads from the current ZIP entry..." to "Reads from the current JAR entry..."

Reply:
Thanks for your review and for spotting this copy&paste error. I've fixed it.